tests/func/test_remote.py

```diff
@@ -49,6 +49,29 @@ def test_relative_path(self):
         config = configobj.ConfigObj(self.dvc.config.files["repo"])
         self.assertEqual(config['remote "mylocal"']["url"], rel)
 
+    def test_remote_modify_validation(self):
+        remote_name = "drive"
+        unsupported_config = "unsupported_config"
+        self.assertEqual(
+            main(["remote", "add", "-d", remote_name, "gdrive://test/test"]), 0
+        )
+        self.assertEqual(
+            main(
+                [
+                    "remote",
+                    "modify",
+                    remote_name,
+                    unsupported_config,
+                    "something",
+                ]
+            ),
+            251,
+        )
+        config = configobj.ConfigObj(self.dvc.config.files["repo"])
+        self.assertFalse(
+            unsupported_config in config['remote "{}"'.format(remote_name)]
+        )
+
```

Comment:
We are in the process of moving to only using pytest style tests (that use regular assert statements), rather than the older class based unittest style. This test can be moved to a standalone function outside of the `TestDvc` class, see the newer tests at the end of this file for more examples

```py
    def test_remote_modify_validation(dvc):
        remote_name = "drive"
        ...
        config = configobj.ConfigObj(dvc.config.files["repo"])
        assert unsupported_config not in config['remote "{}"'.format(remote_name)]

Reply:
Ok